siloLapisTests/test/aminoAcidInsertions.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { expect } from 'chai';
+import { basePath, lapisClient } from './common';
+
+describe('The /aminoAcidInsertions endpoint', () => {
+  let someInsertion = 'ins_S:214:EPE';
+
+  it('should return amino acid insertions for Switzerland', async () => {
+    const result = await lapisClient.postAminoAcidInsertions1({
+      aminoAcidInsertionsRequest: { country: 'Switzerland' },
+    });
+
+    expect(result.data).to.have.length(6);
+
+    const specificInsertion = result.data.find(insertionData => insertionData.insertion === someInsertion);
+    expect(specificInsertion?.count).to.equal(5);
+  });
+
+  it('should order by specified fields', async () => {
+    const ascendingOrderedResult = await lapisClient.postAminoAcidInsertions1({
+      aminoAcidInsertionsRequest: {
+        orderBy: [{ field: 'count', type: 'ascending' }],
+      },
+    });
+
+    expect(ascendingOrderedResult.data[0]).to.have.property('insertion', 'ins_ORF1a:3602:FEP');
+
+    const descendingOrderedResult = await lapisClient.postAminoAcidInsertions1({
+      aminoAcidInsertionsRequest: {
+        orderBy: [{ field: 'count', type: 'descending' }],
+      },
+    });
+
+    expect(descendingOrderedResult.data[0]).to.have.property('insertion', 'ins_S:214:EPE');
+  });
+
+  it('should apply limit and offset', async () => {
+    const resultWithLimit = await lapisClient.postAminoAcidInsertions1({
+      aminoAcidInsertionsRequest: {
+        orderBy: [{ field: 'count', type: 'ascending' }],
+        limit: 2,
+      },
+    });
+
+    expect(resultWithLimit.data).to.have.length(2);
+    expect(resultWithLimit.data[1]).to.have.property('insertion', 'ins_ORF1a:3602:F');
+
+    const resultWithLimitAndOffset = await lapisClient.postAminoAcidInsertions1({
+      aminoAcidInsertionsRequest: {
+        orderBy: [{ field: 'count', type: 'ascending' }],
+        limit: 2,
+        offset: 1,
+      },
+    });
+
+    expect(resultWithLimitAndOffset.data).to.have.length(2);
+    expect(resultWithLimitAndOffset.data[0]).to.deep.equal(resultWithLimit.data[1]);
+  });
+
+  it('should correctly handle nucleotide insertion requests in GET requests', async () => {
+    const result = await lapisClient.postAminoAcidInsertions1({
```

Comment:
This is not a GET request :)
(Same for the test below)

Reply:
I just saw that in the `aggregated.spec.ts`, it's also no GET request where the test claims to send one, seems like something went wrong at some point when changing the code.